src/components/Dialog/Dialog.st.css

```
@@ -0,0 +1,69 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "../IconButton/IconButton.st.css";
+    -st-default: TPAIconButton;
+}
+
+/*Defaults*/
+:vars {
+    DefaultCloseButtonColor: color-5;
+}
+
+/*Overrides*/
+:vars {
+    CloseButtonColor: --overridable;
+}
+
+/*Fallbacks*/
+:vars {
+    CurrentCloseButtonColor: color(fallback(value(CloseButtonColor), value(DefaultCloseButtonColor)));
+}
+
+.root {
+    -st-states: mobile;
+}
+
+.contentWrapper {
+    box-sizing: border-box;
+    min-width: 280px;
+    max-width: 980px;
+    min-height: 240px;
+    max-height: 620px;
+
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -30%);
+
+    background: #fff;
+}
+
+.root:mobile .contentWrapper {
+    height: 100%;
+    transform: translate(-50%, -50%);
```

I didn't notice this before, but maybe it's better to not have translations in mobile?
```suggestion
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    transform: none;
```